qemu/deps/win_serial/serial-host-send.py

```diff
@@ -11,6 +11,7 @@
 
 
 def pack_message(arg):
+    arg = arg.encode('UTF-8')
```

Comment:
Please don't change it like this.
Write a doc str for this function that only bytes is accepted in python3.

change line 37 `with open(data_file, rb) as ff:` to load the file to bytes

Reply:
Updated, thx.